newIDE/app/src/InstancesEditor/index.js

```diff
@@ -59,10 +59,24 @@ export default class InstancesEditorContainer extends Component {
     // if the project changes).
     const { project } = this.props;
 
+    //This prevents flickering on some mobile devices
+    PIXI.glCore.VertexArrayObject.FORCE_NATIVE = true;
+
+    if (project.getScaleMode() === 'nearest') {
+      PIXI.settings.SCALE_MODE = PIXI.SCALE_MODES.NEAREST;
+    } else {
+      PIXI.settings.SCALE_MODE = PIXI.SCALE_MODES.LINEAR;
+    }
+
+    //Create the renderer and setup the rendering area for scene editor.
+    //"preserveDrawingBuffer: true" is needed to avoid flickering and background issues on some mobile phones (see #585 #572 #566 #463)
     this.pixiRenderer = PIXI.autoDetectRenderer(
       this.props.width,
       this.props.height,
-      { antialias: true } //fixes jaggy edges
+      {
+        preserveDrawingBuffer: true,
+        antialias: false,
+      } // Disable antialiasing for instance in scene editor (This affect tiles sprite)
```

Comment:
I would rather write:
"// Disable anti-aliasing to avoid rendering issue (1px width line of extra pixels) when rendering pixel perfect tiled sprites."